### PR TITLE
initial temurin rewrite

### PR DESCRIPTION
**Pull request by gdams**
This PR handles the intial changes required to build Temurin binaries. The CI jobs will likely fail until we have source mirrors at adoptium

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/build-issue.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: 🐛 Bug Report
-about: For problems related to how the adoptopenjdk binaries are created or published
+about: For problems related to how the temurin binaries are created or published
```
Comment (karianna) [inline .github/ISSUE_TEMPLATE/build-issue.md:3]: Capital T

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ about: Suggest a new feature for this project
 labels: 'enhancement'
 ---
 
-Features that impact the whole project (e.g. Adding a new OpenJDK distribution)are made over at the [TSC](https://github.com/AdoptOpenJDK/TSC/).
+Features that impact the whole project (e.g. Adding a new OpenJDK distribution)are made over at the [adoptium](https://github.com/adoptium/adoptium/).
```
Comment (karianna) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]: at the adoptium wg?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Repository for code and instructions for building OpenJDK
+# Repository for code and instructions for building Temurin
 
 [![Slack](https://slackin-jmnmplfpdu.now.sh/badge.svg)](https://slackin-jmnmplfpdu.now.sh/)
 
-AdoptOpenJDK makes use of these scripts to build binaries on the build farm at
+Temurin makes use of these scripts to build binaries on the build farm at
 <https://ci.adoptopenjdk.net>, which produces OpenJDK binaries for consumption via
-<https://adoptopenjdk.net> and <https://api.adoptopenjdk.net>.
+<https://adoptium.net> and <https://api.adoptium.net>.
```
Comment (karianna) [inline README.md:7]: I guess the API isn't up yet...

File: README.md [hunk 4]
```diff
@@ -91,25 +91,25 @@ USAGE
...
-jdk12u - Build Java 12, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk12u
-jdk13u - Build Java 13, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk13u
-jdk - Build Latest Java (Alpha/Beta), defaults to https://github.com/AdoptOpenJDK/openjdk-jdk
-jfx - Build OpenJFX, defaults to https://github.com/AdoptOpenJDK/openjdk-jfx
-amber - Build Project Amber, defaults to https://github.com/AdoptOpenJDK/openjdk-amber
+jdk8u - Build Java 8, defaults to https://github.com/adoptium/openjdk-jdk8u
+jdk9u - Build Java 9, defaults to https://github.com/adoptium/openjdk-jdk9u
+jdk10u - Build Java 10, defaults to https://github.com/adoptium/openjdk-jdk10u
+jdk11u - Build Java 11, defaults to https://github.com/adoptium/openjdk-jdk11u
+jdk12u - Build Java 12, defaults to https://github.com/adoptium/openjdk-jdk12u
```
Comment (karianna) [inline README.md:102]: Do we have all of these mirrored?

File: README.md [hunk 5]
```diff
@@ -302,7 +302,7 @@ Alongside the built assets a metadata file will be created with info about the b
 
 ```json
 {
-    "vendor": "AdoptOpenJDK",
+    "vendor": "Adoptium",
```
Comment (karianna) [inline README.md:305]: Should that be Eclipse Adoptium?

File: build-farm/make-adopt-build-farm.sh [hunk 6]
```diff
@@ -20,7 +20,7 @@ set -e
 PLATFORM_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 ## autodetect defaults to improve usability when running this for debugging/testing
-## On most platforms "uname -p" matches what the OS name used in the adoptopenjdk
+## On most platforms "uname -p" matches what the OS name used in the temurin
```
Comment (karianna) [inline build-farm/make-adopt-build-farm.sh:23]: Temurin

File: makejdk-any-platform.1 [hunk 7]
```diff
@@ -24,36 +24,36 @@ that you are building for further details.
...
 
 .SH VERSIONS
 .TP
 .BR jdk8u
-Build Java 8, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk8u
+Build Java 8, defaults to https://github.com/adoptium/openjdk-jdk8u
 .TP
 .BR jdk9u
-Build Java 9, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk9u
+Build Java 9, defaults to https://github.com/adoptium/openjdk-jdk9u
```
Comment (karianna) [inline makejdk-any-platform.1:38]: Looks like this is out of date versions wise and do we have all of these mirrored?

File: test/functional/buildAndPackage/src/net/adoptium/test/VendorPropertiesTest.java [hunk 8]
```diff
@@ -165,7 +165,7 @@ public void javaVendorVersion(String value) {
 
         @Override
         public void javaVmVendor(String value) {
-            assertTrue(value.equals("AdoptOpenJDK") || value.equals("Eclipse OpenJ9"));
+            assertTrue(value.equals("Adoptium") || value.equals("Eclipse OpenJ9"));
```
Comment (karianna) [inline test/functional/buildAndPackage/src/net/adoptium/test/VendorPropertiesTest.java:168]: Do we still have Eclipse OpenJ9 here - I think it's OK...

**Review by karianna — CHANGES_REQUESTED** [review]
nitpicks

**Comment by gdams** [comment]
@karianna updated PTAL

**Comment by karianna** [comment]
@gdams Linter on Markdown

File: README.md [hunk 9]
```diff
@@ -58,10 +58,10 @@ personally or at build farm scale.
 build jobs used for building Adopt OpenJDK binaries.
 1. The `docker` folder contains tools for generating dockerfiles which can be used as part of building
 OpenJDK inside a Docker container.
-1. The `git-hg` folder has now been moved to it's own seperate repository. See [openjdk-mirror-scripts](https://github.com/AdoptOpenJDK/openjdk-mirror-scripts).
+1. The `git-hg` folder has now been moved to it's own seperate repository. See [openjdk-mirror-scripts](https://github.com/adoptium/mirror-scripts).
 1. The `mercurial-tags/java-tool` folder contains scripts for TODO.
```
Comment (karianna) [inline README.md:62]: I believe this has been deleted

File: sbin/common/constants.sh [hunk 10]
```diff
@@ -51,4 +51,4 @@ export BUILD_VARIANTS="${BUILD_VARIANT_HOTSPOT} ${BUILD_VARIANT_OPENJ9} ${BUILD_
 export GIT_TAGS_TO_SEARCH=100
 
 # Path of marker file in mirror repos
-export ADOPTOPENJDK_MD_MARKER_FILE="AdoptOpenJDK.md"
+export TEMURIN_MD_MARKER_FILE="Temurin.md"
```
Comment (andrew-m-leonard) [inline sbin/common/constants.sh:54]: This change will need doing in conjunction with renaming this in the mirror repos...?

Comment (gdams) [reply sbin/common/constants.sh:54]: This PR switches to build from the adoptium source repos which already have the Temurin.md file https://github.com/adoptium/jdk11u/blob/dev/Temurin.md

File: README.md (hunk 9 above)
Comment (gdams) [reply README.md]: removed

**Pull request merged**